### PR TITLE
Load using cancancan

### PR DESCRIPTION
**Pull request by jcoyne**
## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact (including writing to shared file systems or interacting with other SDR APIs), ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

⚡ ⚠ If this change updates the Argo UI, run all integration tests that use Argo and create a PR on the integration test repo to fix anything this change breaks. ⚡

### CODE REVIEW

**Comment by ndushay** [comment]
um ... why _was_ this change made?  There's no attached ticket, and the title isn't helping me figure that out.

**Comment by jcoyne** [comment]
This lets the gem do the work. We can remove boiler plate code.

File: app/controllers/catkeys_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class CatkeysController < ApplicationController
-  before_action :load_and_authorize_resource
+  load_and_authorize_resource :cocina, parent: false, class: 'Repository', id_param: 'item_id'
```
Comment (justinlittman) [inline app/controllers/catkeys_controller.rb:4]: Can you explain this?

Comment (jcoyne) [reply app/controllers/catkeys_controller.rb:4]: It is a method in the Cancancan gem that we use for authorization: https://github.com/CanCanCommunity/cancancan/blob/develop/docs/controller_helpers.md#authorize_resource-load_resource-load_and_authorize_resource

Comment (justinlittman) [reply app/controllers/catkeys_controller.rb:4]: Thanks. Ruby magic.

**Review by justinlittman — APPROVED** [review]

**Pull request merged**